task_web/app/controllers/admin/users_controller.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+class Admin::UsersController < ApplicationController
```

Comment:
Rubocopを通してみたところ以下の警告が出ていました
> app/controllers/admin/users_controller.rb:3:7: C: Style/ClassAndModuleChildren: Use nested module/class 
> definitions instead of compact style.
> class Admin::UsersController < ApplicationController
> 　　　^^^^^^^^^^^^^^^^^^^^^^
moduleの例ですが、以下のページに推奨されない理由がありました。
https://journal.sooey.com/242
ラクマのクラスも階層化するスタイルで統一されているので修正をお願いしますー
↓ラクマ内での階層化されているクラスたちの例
https://github.com/Fablic/fril-engine/tree/master/app/models/fril

Reply:
他の警告は対応が難しそうなので無視してOKだとおもいます